tools/azsdk-cli/Azure.Sdk.Tools.Cli/Helpers/UserHelper.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Azure.Identity;
+using Microsoft.Graph;
+
+namespace Azure.Sdk.Tools.Cli.Helpers
+{
+    public interface IUserHelper
+    {
+        public Task<string> GetUserEmail();
+    }
+    public class UserHelper: IUserHelper
+    {
+        private readonly string[]  scopes = new[] { "https://graph.microsoft.com/.default" };
+        private readonly DefaultAzureCredential credential = new DefaultAzureCredential();
```

Comment:
And we don't need to make this lazy because if the environment isn't configured properly, this only will error if we _use_ it right?

Reply:
It uses graph call only when GetUserEmail is called. So it's still lazy call. I can move that creds to method call to make it more clear.